rosdep/base.yaml

```diff
@@ -602,6 +602,17 @@ dnsmasq:
   fedora: [dnsmasq]
   gentoo: [net-dns/dnsmasq]
   ubuntu: [dnsmasq]
+docker:
```

Comment:
If you don't need the System Tray docklet which is installed via the `docker` package on Debian and Ubuntu this key can be removed.

Reply:
This does look like it's unrelated.